schema/defs.json

```diff
@@ -29,6 +34,21 @@
                 },
                 "permissions": {
                     "type": "string"
+                },
+                "type": {
+                    "type": "string"
+                },
+                "major": {
+                    "$ref": "#/devinitions/int64"
```

Comment:
definitions (for the next 3 items below too)

Reply:
done